repository/src/test/java/org/jbpm/process/workitem/repository/service/RepoServiceIntegrationTest.java

```diff
@@ -151,21 +153,24 @@ public void testAddServiceTask(){
 
     @Test
     public void testRemoveServiceTask() {
-        Optional<RepoData> beforeRemoveOne = repoService.getServices().stream().filter(s -> s.getName().equals(SERVICE_TASK_NAME_ONE)).findFirst();
+        assertTrue(getInitRepoDataList().size() > 2);
+        String service_task_name_one = getInitRepoDataList().get(0).getName();
```

Comment:
Naming of the variables.

Reply:
What convention for naming is preferred? Could we fix it to not block the merge or do it later?

Reply:
In Java? Camel case.
service_task_name_one -> serviceTaskNameOne ...

Reply:
Done